pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -212,6 +221,11 @@ type VolumeSnapshotClassList struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:scope=Cluster
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="DeletionPolicy",type=string,JSONPath=`.spec.deletionPolicy`,description="Determines whether a VolumeSnapshotContent created through the VolumeSnapshotClass should be deleted when its bound VolumeSnapshot is deleted."
+// +kubebuilder:printcolumn:name="Driver",type=string,JSONPath=`.spec.driver`,description="Name of the CSI driver used to create the physical snapshot on the underlying storage system."
+// +kubebuilder:printcolumn:name="VolumeSnapshotClass",type=string,JSONPath=`.spec.volumeSnapshotClassName`,description="Name of the VolumeSnapshotClass to which this snapshot belongs."
+// +kubebuilder:printcolumn:name="VolumeSnapshot",type=string,JSONPath=`.spec.volumeSnapshotRef.name`,description="The VolumeSnapshot object to which this VolumeSnapshotContent object is bound."
+// +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`
```

Comment:
s/Age/CreationTime

Reply:
I think CreationTime, RestoreSize, and ReadyToUse fields in Status are all useful to be printed out.

Reply:
I've left the column title as `Age` to be consistent with the rest of the `get` outputs.

RestoreSize and RestoreTime have been included in the output. I've left CreationTime off, as this is an int64 and not a metav1.Time. Attempting to print it out as a date looks different than the other outputs, so I've left it off for now.